sw/device/lib/base/macros.h

```diff
@@ -443,4 +451,57 @@
  */
 #define OT_ALIGN_MEM(x) (uint32_t)(4 + (((uintptr_t)(x)-1) & ~3))
 
+#ifndef RUST_PREPROCESSOR_EMIT
```

Comment:
Ah, both of these definitions probably need `#if !defined(__ASSEMBLER__) && !defined(NOSTDINC)` because test_rom_start.S includes macros.h.